EAHW-2813 - Update readme - use within LocalDev (#26)
UKHomeOffice/callisto-auth-keycloak: README.md

- Merges into main trigger a workflow that applies the terraform configuration
- A nightly workflow runs a terraform plan with a detailed exit code which causes a failure if any changes are detected

## Using Keycloak locally

> It is suggested to run Keycloak from within [callisto-localdev solution](https://github.com/UKHomeOffice/callisto-localdev). When used like that, it doesn't need any configuration to start development. However, it is possible to run Keycloak manually (check below section).

Reviewer: This is really picky but I didn't like the wording
```suggestion
It is suggested to run Keycloak from within [callisto-localdev solution](https://github.com/UKHomeOffice/callisto-localdev). When run within the localdev solution, it doesn't need any configuration to start development. However, it is possible to run Keycloak manually (check below section).
```

Access key and secret key for the terraform s3 bucket can be found in Kubernetes secrets.

You will also need to get the client secret (found in the credentials tab) from the terraform-client in the Callisto

> realm in keycloak, and update the `KEYCLOAK_CLIENT_SECRET` in Drone.

Reviewer: Do we need the all Callisto repos section?

Author: added now
